RPG Wizardry/Assets/Scripts/Player/Inventory/PlayerInventory.cs

```diff
@@ -23,6 +23,14 @@ public class PlayerInventory : MonoBehaviour, IStorable, IJSON<PlayerInventory>
         #region Variables
         #region Public
         /// <summary>
+        /// Creating event array to check for spellunlocking
+        /// </summary>
+        public delegate void SpellUnlock();
+        /// <summary>
+        /// Spellunlocked event
+        /// </summary>
+        public static SpellUnlock spellunlocked;
```

Comment:
Should not be static. PlayerInventory can be reference via PlayerManager.Instance